Omitting --version on api create puts project in broken state

I made a boo boo when typing but it uncovered a bug:

Steps to reproduce
```
kubebuilder init --domain localytics.com --license none
kubebuilder create api --group bigthicket --kind EphemeralEnvironment --namespaced false
```

Output
```
Create Resource under pkg/apis [y/n]?
y
Create Controller under pkg/controller [y/n]?
y
Writing scaffold for you to edit...
pkg/apis/bigthicket/ephemeralenvironment_types.go
pkg/apis/bigthicket/ephemeralenvironment_types_test.go
/*
.
*/

// NOTE: Boilerplate only.  Ignore this file.

// Package  contains API Schema definitions for the bigthicket  API group
// +k8s:openapi-gen=true
// +k8s:deepcopy-gen=package,register
// +k8s:conversion-gen=github.com/localytics/big-thicket/controller/pkg/apis/bigthicket
// +k8s:defaulter-gen=TypeMeta
// +groupName=bigthicket.localytics.com
package 

import (
	"k8s.io/apimachinery/pkg/runtime/schema"
	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
)

var (
	// SchemeGroupVersion is group version used to register these objects
	SchemeGroupVersion = schema.GroupVersion{Group: "bigthicket.localytics.com", Version: ""}

	// SchemeBuilder is used to add go types to the GroupVersionKind scheme
	SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
)

2018/07/19 20:30:49 pkg/apis/bigthicket/register.go:15:1: expected 'IDENT', found 'import'
```

Subsequent attempts to do anything, such as re-create the resource with the right flags, end up with:

```
can't load package: package github.com/localytics/big-thicket/controller/pkg/apis/bigthicket: 
pkg/apis/bigthicket/register.go:1:1: expected 'package', found 'EOF'
W0719 20:31:32.397711   16348 parse.go:239] Ignoring child directory github.com/localytics/big-thicket/controller/pkg/apis/bigthicket: unable to import "github.com/localytics/big-thicket/controller/pkg/apis/bigthicket": /Users/bwarminski/go/src/github.com/localytics/big-thicket/controller/pkg/apis/bigthicket/register.go:1:1: expected 'package', found 'EOF'
make: *** [generate] Error 1
2018/07/19 20:31:32 exit status 2
```

I think I saw a validation check running around, but maybe it happens too late in the process and empty files were scaffolded. Happy to attempt a fix once I've gotten some comfort with the happy path.

--- a/cmd/kubebuilder/v1/api.go
+++ b/cmd/kubebuilder/v1/api.go
@@ -50,9 +50,20 @@ func (o *apiOptions) RunAddAPI() {
 		fmt.Println("Create Resource under pkg/apis [y/n]?")
 		o.doResource = util.Yesno(reader)
 	}
+
 	if !o.controllerFlag.Changed {
 		fmt.Println("Create Controller under pkg/controller [y/n]?")
 		o.doController = util.Yesno(reader)
+	}
+
+	if o.r.Group == "" {
+		log.Fatalf("Must specify --group")
+	}
+	if o.r.Version == "" {
+		log.Fatalf("Must specify --version")
+	}
+	if o.r.Kind == "" {
+		log.Fatalf("Must specify --kind")
 	}
 
 	fmt.Println("Writing scaffold for you to edit...")

